docs/use-cases-orchestration.md

```diff
@@ -4,15 +4,57 @@ title: Microservice Orchestration and Saga
 ---
 
-It is common that some business processes are implemented as multiple microservice calls.
-And the implementation must guarantee that all of the calls must eventually succeed even with the occurrence of prolonged downstream service failures.
-In some cases, instead of trying to complete the process by retrying for a long time, compensation rollback logic should be executed.
-[Saga Pattern](https://microservices.io/patterns/data/saga.html) is one way to standardize on compensation APIs.
+## Motivation
 
-Temporal is a perfect fit for such scenarios. It guarantees that workflow code eventually completes, has built-in support
-for unlimited exponential activity retries and simplifies coding of the compensation logic. It also gives full visibility into the state of each workflow, in contrast to an orchestration based on queues where getting a current status of each individual request is practically impossible.
+It is common to break a large application into **microservices** structured around the distinct business capabilities. Each microservice typically has its storage and interacts with other services via an API. At the same time, business processes may need to call multiple microservices to achieve the desired outcome.
 
-Following are some real-world examples of Temporal-based service orchestration scenarios:
+The implementation must guarantee that all of the calls must eventually succeed even with the occurrence of prolonged failures of individual microservices.
 
- * [Using Temporal workflows to spin up Kubernetes by Banzai Cloud](https://banzaicloud.com/blog/introduction-to-cadence/)
- * [Improving the User Experience with Uber’s Customer Obsession Ticket Routing Workflow and Orchestration Engine](https://eng.uber.com/customer-obsession-ticket-routing-workflow-and-orchestration-engine/)
+In some cases, instead of trying to complete the process by retrying for a long time, compensation rollback logic should be executed. [Saga Pattern](https://microservices.io/patterns/data/saga.html) is one way to standardize compensation APIs.
+
+## Benefits
+
+Temporal is a perfect fit for such scenarios. It guarantees that workflow code eventually completes, has built-in support for unlimited exponential activity retries, and simplifies the coding of the compensation logic. 
+
+Temporal also gives full visibility into the state of each workflow, in contrast to an ad-hoc orchestration based on queues where getting a current status of each request is virtually impossible.
+
+## Example
+
+The workflow snippet below orchestrates two activity calls: booking a hotel and reserving a flight. If the first activity fails (including all the configured retries), the workflow returns directly.
+
+However, if the first reservation succeeds, but the second one fails, you may want to cancel the hotel to avoid undesired charges. That's why the error-handling block contains a third activity call that cancels the hotel reservation before completing the workflow.
```

Comment:
```suggestion
However, if the first reservation succeeds, but the second one fails, you may want to cancel the already booked hotel to avoid undesirable charges. This is why the error-handling block contains a third activity call which cancels the hotel reservation before completing the workflow.
```